### DISCUSSION TITLE
Unable to connect to MQTT

### DISCUSSION THREAD

**Discussion by paducz** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#808] [General]
Hi, can you please confirm that MQTT is still working? After updating to the latest firmware I am no more able to connect to MQTT broker. I do not know where is the problem.

**Comment by visualapproach** [comment]
Yes it is working. Perhaps you need to update the parameters on the MQTT settings page?
It will take a minute before it connects to the mqtt.
What is the status of the connection? See bottom of main page

**Reply by paducz** [reply]
Thank you for prompt reply. The status is CONNECT_FAILED...I have home assistant with mosquito mqtt...I am not sure which IP address should i enter...when entering ip of the home assistant. it is not working

**Reply by visualapproach** [reply]
Yes the ip is the ip of homeassistant. Check username and password. You can create a new user in HA without messing with the mqtt broker. Also check logs in HA.
Download MQTT Explorer on PC and see if you can connect to the broker using the same credentials. If not, the problem is in your HA/MQTT setup.

**Reply by paducz** [reply]
Will try the mqtt explorer later. Client ID and base topic can whatever?

**Reply by paducz** [reply]
I tried everything, MQTT not working enymore for me. I will try to downgrade firmware next days and will see

**Reply by paducz** [reply]
Are there some logs so I can check why it is failing?

**Reply by paducz** [reply]
Btw I am not sure if it is normal, but GUI does not load the MQTT values I entered

**Reply by visualapproach** [reply]
Make sure that the version of the filesystem on the esp matches the fw version. I.e. don't forget to update both the fw and the files.
Then remember to click on save and wait a short while before rebooting

**Reply by paducz** [reply]
I noticed I am getting this error during file system upload via OTA, board D1 mini PRO (unfortunately I am not able to do update via wire at this moment)
Uploading: [============================================================] 100% Done...


12:59:27 [ERROR]: Error Uploading
*** [uploadfsota] Error 1
==================================================== [FAILED] Took 24.37 seconds ===========

**Reply by visualapproach** [reply]
Dont click the upload OTA. Use upload file system

**Reply by paducz** [reply]
Works! Thank you for your support
